package.json

```diff
@@ -43,6 +43,7 @@
         "react-dom": "^16.8.2",
         "react-dropdown": "^1.6.4",
         "react-helmet": "^5.2.0",
+        "react-inlinesvg": "^0.8.4",
         "react-redux": "^6.0.0",
```

Comment:
Do we need the redux package?

Reply:
It's not yet used, I'm planning to have a go at it next as this state management are becoming crazy.